util/EIP-712/src/lib.rs

```diff
@@ -156,35 +156,22 @@
 //! }
 //! ```
 
-#![warn(missing_docs, unused_extern_crates)]
+#![warn(missing_docs)]
```

Comment:
Why remove `unused_extern_crates`?

Reply:
this module uses rust 2018, no need for extern crates